install/tutorial_12.md

```diff
@@ -0,0 +1,252 @@
+# Download and Install SDFormat
+
+## Binary Installation
+
+### Ubuntu
+
+On Ubuntu systems, `apt-get` can be used to install `sdformat`:
+
+    sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
+    wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
+    sudo apt-get update
+
+    sudo apt install libsdformat<#>-dev libsdformat<#>
+
+Be sure to replace `<#>` with a number value, such as 9 or 10, depending on
+which version you need.
+
+### macOS
+
+On macOS, add OSRF packages:
+
+    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+    brew tap osrf/simulation
+
+Install sdformat:
+
+    brew install sdformat<#>
+
+Be sure to replace `<#>` with a number value, such as 9 or 10, depending on
+which version you need.
+
+### Windows
+
+Install [Conda package management system](https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html).
+Miniconda suffices.
+
+Create if necessary, and activate a Conda environment:
+
+    conda create -n ign-ws
+    conda activate ign-ws
+
+Install `sdformat`:
+
+    conda install libsdformat --channel conda-forge
+
+You can view all the versions with
+
+    conda search libsdformat --channel conda-forge
+
+and install a specific minor version with
+
+    conda install libsdformat=9.3.0 --channel conda-forge
+
+## Source installation
+
+### Ubuntu
+
+#### Prerequisites
+
+Make sure you have removed the Ubuntu pre-compiled binaries before installing from source
+
+    sudo apt-get remove 'libsdformat*' 'sdformat*'
+
+If you have previously installed from source, be sure you are installing to the same path location or that you have removed the previous installation from source version manually.
+
+#### Install Required Dependencies
+
+Install prerequisites. A clean Ubuntu system will need:
+
+##### build tools, ruby for building xml schemas, tinyxml, and boost:
+
+    sudo apt -y install \
+      $(sort -u $(find . -iname 'packages-'`lsb_release -cs`'.apt' -o -iname 'packages.apt' | grep -v '/\.git/') | tr '\n' ' ')
+
+#### Install Optional Dependencies
+
+##### USD
+
```

Comment:
For those who don't know what USD is, it would be great to have a sentence describing what it is with a link to the official website.